packages/framework/src/Resources/config/services_test.yml

```diff
@@ -172,7 +180,7 @@ services:
 
     Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade: ~
 
-    Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportRepository: ~
```

Comment:
Hello, isn't this a BC break ? cause my tests started to fail.

Reply:
Hi, Thanks for noticing. I created a new PR (#1006) to fix this